src/js/index.js

```diff
@@ -492,10 +504,12 @@ export const hpe = deepFreeze({
   select: {
     icons: {
       color: 'text',
+      down: FormDown,
+      up: FormUp,
     },
     options: {
       text: {
-        size: 'small',
+        size: 'medium',
```

Comment:
If we leave this undefined, will it use the default of medium?

Reply:
That would be my expectation, but didn't explicitly check.

Reply:
Cool, just wondering if we don't want to add unnecessary values to the theme if it's covered by default vs being exhaustive

Reply:
I agree it's unnecessary to state the object unless it overrides the base.js styles.

Reply:
removed.